### PR TITLE
Update redis_geo.cc

### PR DESCRIPTION
**Pull request by byronhe**

### CODE REVIEW

File: src/redis_geo.cc [hunk 1]
```diff
@@ -174,7 +174,7 @@ rocksdb::Status Geo::MGet(const Slice &user_key,
 }
 
 std::string Geo::EncodeGeoHash(double longitude, double latitude) {
-  std::string geoalphabet = "0123456789bcdefghjkmnpqrstuvwxyz";
+  const static std::string geoalphabet = "0123456789bcdefghjkmnpqrstuvwxyz";
```
Comment (git-hulk) [inline src/redis_geo.cc:177]: CI failed, `static` identifier is unnecessary

**Review by git-hulk — CHANGES_REQUESTED** [review]

File: src/redis_geo.cc (hunk 1 above)
Comment (ShooterIT) [reply src/redis_geo.cc:177]: > src/redis_geo.cc:177:`  Storage-class specifier (static, extern, typedef, etc) should be at the beginning of the declaration.

@git-hulk From cpplint detail, it seems we should change `const static` into `static const`?

Comment (git-hulk) [reply src/redis_geo.cc:177]: yeah, or remove the `static` should be OK

Comment (ShooterIT) [reply src/redis_geo.cc]: Hi @git-hulk @byronhe Since we haven't handle with it for a long time, i just remove `static` to resolve this patch

**Review by ShooterIT — APPROVED** [review]

**Review by git-hulk — APPROVED** [review]

**Pull request merged**